src/shared/functions.py

```diff
@@ -139,7 +139,10 @@ def update_metadata(
             data_version_metadata[target] = dt.isoformat()
         case enums.JsonFilename.COLUMN_TYPES.value:
             study_metadata = metadata.setdefault(study, {})
-            data_package_metadata = study_metadata.setdefault(data_package, {})
+            if extra_items.get("type") == "flat":
+                data_package_metadata = study_metadata.setdefault(f"{data_package}__{site}", {})
+            else:
+                data_package_metadata = study_metadata.setdefault(data_package, {})
```

Comment:
Why do we just add site to the name for flat files? Would cube files also benefit from it?

Reply:
so for flat files, since we're showing one instance per site, we need to get the site name into the 4th subfolder of the s3 path, which is what the glue crawler uses to generate the table name.

for cubes, since we don't need site-based tables, we can skip it, and let aggregation handle it.